### ISSUE TITLE
Loss and Metrics Implementation

### ISSUE THREAD

**Issue by teamwong111** [Tiexin-RS/segment-with-nn#5]
- [ ] loss
  - [ ] focal loss
  - [ ] losavz softmax
  - [ ] cross entropy loss
  - [ ] dice loss
  - [ ] boundy_loss
- [ ] metrics
  - [ ] IOU
